com.ibm.wala.core/src/com/ibm/wala/analysis/typeInference/TypeInference.java

```diff
@@ -727,8 +727,8 @@ public void visitStoreIndirect(SSAStoreIndirectInstruction instruction) {
 
     @Override
     public IVariable makeVariable(int valueNumber) {
-      SymbolTable st = ir.getSymbolTable();
       if (doPrimitives) {
+        SymbolTable st = ir.getSymbolTable();
```

Comment:
Are these changes to `TypeInference` part of the fix for default methods?  If not, can we pull all changes to `TypeInference` into a separate PR, to keep things clean?